inmaputil/cmd.go

```diff
@@ -569,6 +575,33 @@ func init() {
 	Root.AddCommand(srCmd)
 	srCmd.AddCommand(srPredictCmd)
 	Root.AddCommand(workerCmd)
+
+	// For some reasons, HTTPPort flag never seem to be passed down?
+	// HTTPPort always uses default value (8080), it's okay for now
+	if Cfg.IsSet("HTTPPort") && Cfg.GetString("HTTPPort") != "" { 
+		fmt.Println("Loading front-end...")
+
+		output := template.Must(template.New("").ParseFiles("../inmaputil/wrapper.html"))
```

Comment:
When wrapper.html is a separate file, the program will only work when it is in the same directory as wrapper.html. Instead, we can include the wrapper html as a text string within the go source code, e.g., 
``` go
const wrapper = `doctype html etc...`
```

Reply:
Alright, I will do that. This will assume that you are OK with the long wrapper.html being in the source code.